gdk-pixbuf/sys/Gir.toml

```diff
@@ -25,17 +25,13 @@ name = "GdkPixbuf.Pixbuf"
 status = "generate"
```

Comment:
The whole config of GdkPixbuf.Pixbuf can be dropped.

Reply:
Okay. so does that mean everything in the `gdk-pixbuf/src/sys`, `gdk-pixbuf/src/sys` and `gdk-pixbuf/Gir.toml` should go or something else like a single file?

Reply:
No, where did you read that? Only the config for `GdkPixbuf.Pixbuf` and `PixbufAnimation` should be dropped as they're empty now so the default handling would do the correct thing.

By removing the `is_windows_utf8 = true` from all the function configurations you made them no-op configurations.